docs/src/BenchImages.tsx

```diff
@@ -0,0 +1,170 @@
+import image057 from '../public/static/branding/mui-x/chart-light3.png';
+import image058 from '../public/static/branding/mui-x/chart-light4.png';
+import image059 from '../public/static/branding/mui-x/Mocktable-light.png';
+import image050 from '../public/static/social-previews/default-preview.jpg';
+import image061 from '../public/static/themes/onepirate/appFooterFacebook.png';
+import image062 from '../public/static/themes/onepirate/appFooterTwitter.png';
+import image063 from '../public/static/themes/onepirate/productHeroWonder.png';
+import image064 from '../public/static/themes/onepirate/productHeroArrowDown.png';
+import image065 from '../public/static/themes/onepirate/productCurvyLines.png';
+import image066 from '../public/static/themes/onepirate/productHowItWorks1.svg';
+import image067 from '../public/static/themes/onepirate/productHowItWorks2.svg';
+import image068 from '../public/static/themes/onepirate/productHowItWorks3.svg';
+import image069 from '../public/static/themes/onepirate/producBuoy.svg';
+import image060 from '../public/static/themes/onepirate/productValues1.svg';
+import image071 from '../public/static/themes/onepirate/productValues2.svg';
+import image072 from '../public/static/themes/onepirate/productValues3.svg';
+import image073 from '../public/static/branding/mui-x/sparkline-dark1.png';
+import image074 from '../public/static/branding/mui-x/sparkline-dark2.png';
+import image075 from '../public/static/branding/mui-x/chart-dark1.png';
+import image076 from '../public/static/branding/mui-x/chart-dark2.png';
+import image077 from '../public/static/branding/mui-x/chart-dark3.png';
+import image078 from '../public/static/branding/mui-x/chart-dark4.png';
+import image079 from '../public/static/branding/mui-x/Mocktable-dark.png';
+
+const images = [
+  image001,
+  image002,
+  image003,
+  image004,
+  image005,
+  image006,
+  image007,
+  image008,
+  image009,
+  image010,
+  image011,
+  image012,
+  image013,
+  image014,
+  image015,
+  image016,
+  image017,
+  image018,
+  image019,
+  image021,
+  image022,
+  image023,
+  image024,
+  image025,
+  image026,
+  image027,
+  image028,
+  image029,
+  image020,
+  image031,
+  image032,
+  image033,
+  image034,
+  image035,
+  image036,
+  image037,
+  image038,
+  image039,
+  image030,
+  image041,
+  image042,
+  image043,
+  image044,
+  image045,
+  image046,
+  image047,
+  image048,
+  image049,
+  image040,
+  image051,
+  image052,
+  image053,
+  image054,
+  image055,
+  image056,
+  image057,
+  image058,
+  image059,
+  image050,
+  image061,
+  image062,
+  image063,
+  image064,
+  image065,
+  image066,
+  image067,
+  image068,
+  image069,
+  image060,
+  image071,
+  image072,
+  image073,
+  image074,
+  image075,
+  image076,
+  image077,
+  image078,
+  image079,
+];
+
+export default function BenchImages() {
+  return (
+    <div>
+      {images.map((image) => (
+        <img {...image} alt="" />
```

Comment:
I thought you had to use `Image` component from `next/Image` to benefits from next optimization

``` suggestion
        <Image {...image} alt="" />
```

Reply:
Yes, this can be used with the `Image` component in Next.js. This PR is mostly about finding out whether the static build of these image files keeps working across the usage of the monorepo dependency in X and Toolpad. 

It'd need to be used as 

```suggestion
        <Image src={image} alt="" />
```

Next.js transforms the import of images to an object containing `src`, `wisth` and `height`, that's why this works with `img` as well. (src contains the url to a static asset that is also included in the build output)

Reply:
Ok, I did not get how you were testing on X and Toolpad, but now I understand. The idea is to test if the doit logo is still accessible, which is the case 🎉

![image](https://user-images.githubusercontent.com/45398769/215488629-951653c2-f917-4845-89c8-bf4ec37cc0e0.png)

Reply:
Yes, I created PRs on X and Toolpad repos and install this branch in them to see how it would behave if we were to merge this PR.


https://github.com/mui/mui-x/pull/6936
https://github.com/mui/mui-toolpad/pull/1367